src/libasr/pass/intrinsic_functions.h

```diff
@@ -568,6 +569,64 @@ namespace ObjectType {
 
 } // namespace ObjectType
 
+namespace Input {
```

Comment:
Since you are implementing support for `input` at the ASR level, we would not be needing this `Input` in the intrinsic registry. This will be redundant.